client/ionic/src/pages/Menu.module.css

```diff
@@ -0,0 +1,4 @@
+.page ion-header ion-toolbar {
+  --background: translucent;
+  --background: blue;
```

Comment:
ditto

Reply:
removed in https://github.com/WorldHealthOrganization/app/pull/74